src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs

```diff
@@ -0,0 +1,138 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+
+namespace System.Diagnostics.Tracing
+{
+    /// <summary>
+    /// RuntimeEventSource is an EventSource that represents events emitted by the managed runtime.
+    /// </summary>
+    [EventSource(Name = "System.Runtime")]
+    internal sealed class RuntimeEventSource : EventSource
+    {
+        internal static RuntimeEventSource m_RuntimeEventSource;
```

Comment:
Nit: The coding convention is to prefix the static fields with `s_`.

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md